src/Function.cpp

```diff
@@ -739,8 +868,27 @@ const std::vector<std::string> Function::args() const {
     return arg_names;
 }
 
+const std::vector<std::string> Function::all_args() const {
```

Comment:
Since this (and the next) method are returning-by-value, is there any point to making the return value const?